src/main/java/com/tw/go/plugin/util/StringUtils.java

```diff
@@ -4,4 +4,24 @@
     public static boolean isEmpty(String str) {
         return str == null || str.trim().isEmpty();
     }
+
+    /**
+     * Extract the group/project resp. organisation/repository path for GitLab and GitHub.
+     *
+     * @param url - "git@github.com:mfriedenhagen/gocd-build-status-notifier.git"
+     * @return - "mfriedenhagen/gocd-build-status-notifier"
+     */
+    public static String getRepository(String url) {
```

Comment:
Hm, there is no RepositoryUtils right now, seems a bit of the top for one method :grinning: